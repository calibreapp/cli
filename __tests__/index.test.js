describe('Index', () => {
  test('should be able to resolve the same dependencies', () => {
    expect(require('../')).toMatchSnapshot()
  })
})
