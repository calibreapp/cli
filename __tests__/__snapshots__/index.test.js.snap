// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should be able to resolve the same dependencies 1`] = `
Object {
  "Agent": Object {
    "list": [Function],
  },
  "Site": Object {
    "create": [Function],
    "destroy": [Function],
    "list": [Function],
  },
  "Snapshot": Object {
    "create": [Function],
    "list": [Function],
  },
  "Test": Object {
    "create": [Function],
    "getList": [Function],
    "getTestByUuid": [Function],
    "waitForTest": [Function],
  },
  "TestProfile": Object {
    "create": [Function],
    "destroy": [Function],
    "list": [Function],
    "update": [Function],
  },
}
`;
